pkg/controllers/disruption/expiration_test.go

```diff
@@ -76,6 +76,47 @@ var _ = Describe("Expiration", func() {
 		})
 		nodeClaim.StatusConditions().MarkTrue(v1beta1.Expired)
 	})
+	Context("Metrics", func() {
+		const eligibleNodesMetric = "karpenter_disruption_eligible_nodes"
+		var eligibleNodesLabels = map[string]string{
+			"method":             "expiration",
+			"consolidation_type": "",
+		}
+		It("should correctly report eligible nodes", func() {
+			pod := test.Pod(test.PodOptions{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						v1beta1.DoNotDisruptAnnotationKey: "true",
+					},
+				},
+			})
+			ExpectApplied(ctx, env.Client, nodePool, nodeClaim, node, pod)
+			ExpectManualBinding(ctx, env.Client, pod, node)
+
+			// inform cluster state about nodes and nodeclaims
+			ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx, env.Client, nodeStateController, nodeClaimStateController, []*v1.Node{node}, []*v1beta1.NodeClaim{nodeClaim})
+
+			fakeClock.Step(10 * time.Minute)
+			wg := sync.WaitGroup{}
+			ExpectTriggerVerifyAction(&wg)
+			ExpectReconcileSucceeded(ctx, disruptionController, types.NamespacedName{})
+			wg.Wait()
+
+			ExpectMetricGaugeValue(eligibleNodesMetric, 0, eligibleNodesLabels)
+
+			// remove the do-not-disturb annotation to make the node eligible for drift and update cluster state
```

Comment:
```suggestion
			// remove the do-not-disrupt annotation to make the node eligible for drift and update cluster state
```

Reply:
You un-did the change!

Reply:
I'll redo it...did a rebase and force push but had forgot to pull first.